examples/image_io/src/main.rs

```diff
@@ -0,0 +1,127 @@
+use anyhow::Result;
+use clap::Parser;
+use kornia_image::Image;
+use kornia_io::{
+    read_image_any_rgb8,
+    write_image_png_gray8, write_image_png_rgb8, write_image_png_rgba8,
+};
+
+#[cfg(feature = "turbojpeg")]
+use kornia_io::{
+    write_image_jpegturbo_gray8, write_image_jpegturbo_rgb8,
+};
+
+/// Command line arguments for the image_io example
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    /// Path to the input image file
+    #[arg(short, long)]
+    input_path: String,
+
+    /// Path where the output image will be saved
+    #[arg(short, long)]
+    output_path: String,
+
+    /// Output format (png or jpeg)
+    #[arg(short, long, default_value = "png")]
+    format: String,
+
+    /// Number of channels for the output image (1 for grayscale, 3 for RGB, 4 for RGBA - PNG only)
+    #[arg(short, long, default_value_t = 3)]
+    channels: usize,
+}
+
+fn main() -> Result<()> {
+    // Parse command line arguments
+    let args = Args::parse();
+
+    // Validate input format
+    if !["png", "jpeg"].contains(&args.format.as_str()) {
+        anyhow::bail!("Unsupported format: {}. Only 'png' and 'jpeg' are supported.", args.format);
```

Comment:
use something like `return Err(format!"Unsupported format: {}. Only 'png' and 'jpeg' are supported.", args.format).into())`